### ISSUE TITLE
Bug: no-unused-expressions nested if

### ISSUE THREAD

**Issue by helixbass** [helixbass/eslint-plugin-coffee#17]
This appears to be a bug that `no-unused-expressions` is triggering on this:
```
x = (if true
  unless false
    if false then false else
      if true
        nonce)
```
